### DISCUSSION TITLE
支持设置youtube字幕样式

### DISCUSSION THREAD

**Discussion by devyujie** [immersive-translate/immersive-translate#1076] [Ideas]
### 新功能 | New Feature

1.支持设置字幕字体，大小，背景

### 为什么您想要添加这些功能 | Why you want to add these features

1.更美观

**Comment by theowenyoung** [comment]
暂时无此计划，目前直接使用 Youtube 默认的样式。

也许这个issue 可以给 youtube 提一下？

**Comment by theowenyoung** [comment]
![](https://s.immersivetranslate.com/assets/youtube-subtitle-options.jpeg)

好吧，原来 Youtube 本身就是支持字幕样式的，那直接在Youtube 字幕设置的地方设置即可～
